Client/game_sa/CPedSA.cpp

```diff
@@ -94,7 +88,7 @@ void CPedSA::Init()
     CPedIntelligenceSAInterface* m_pPedIntelligenceInterface = (CPedIntelligenceSAInterface*)(dwPedIntelligence);
     m_pPedIntelligence = new CPedIntelligenceSA(m_pPedIntelligenceInterface, this);
     m_pPedSound = new CPedSoundSA(&pedInterface->pedSound);
-    m_pDefaultPedSound = new CPedSoundSA(&pedInterface->pedSound);
+    m_pDefaultPedSound = new CPedSoundSA(std::move(pedInterface->pedSound));
```

Comment:
What exactly are you trying to move here? CPedSoundSA constructor below doesn't implement any move.

Reply:
`std::move` transforms pedSound into a rvalue. pedSound need to be a copy and by `std::move`ing it, it kinda achieves it but I could've better written it now that I think of it

Reply:
This is counterintuitive and breaks principles of the C++ standard.